content/geometry/object-vehicle/vehicle-structure.adoc

```diff
@@ -1610,14 +1788,50 @@ This group typically does not move in the simulation and therefore does not have
 2+^| <Header>
 
 | *Origin*
-| Origin of the vehicle (part)
+| Origin of the vehicle
 
 | *x-axis*
-| Collinear with the vehicle longitudinal axis, pointing forwards
+| Collinear with the vehicle's longitudinal axis, pointing forwards
 
 | *y-axis*
 | Completes the right-handed coordinate system
 
 | *z-axis*
 | Perpendicular to the x-axis, pointing vertically upwards
 |===
+
+
+== Grp_Vehicle_Part
```

Comment:
I would propose to leave it in, because then people can already gather experience in that direction and bring that into the project for the next OpenMATERIAL version.
But I don't have a strong opinion about it.

Reply:
You need to approve this PR, otherwise I cannot merge, because your change request is still open.

Reply:
Never mind, I could dismiss your change request. Now I can merge.